src/NodeAutoCompleteViewExtension/ViewModels/NodeAutoCompleteBarViewModel.cs

```diff
@@ -191,13 +191,20 @@ private void ReAddNode(int index)
         internal void ConsolidateTransientNodes()
         {
             var node = PortViewModel.NodeViewModel;
+
+            //unlock undo/redo
+            node.WorkspaceViewModel.Model.UndoRedoLocked = false;
+            
             var transientNodes = node.WorkspaceViewModel.Nodes.Where(x => x.IsTransient).ToList();
             foreach (var transientNode in transientNodes)
             {
                 transientNode.IsTransient = false;
             }
 
             NodeAutoCompleteUtilities.PostAutoLayoutNodes(node.WorkspaceViewModel.Model, node.NodeModel, transientNodes.Select(x => x.NodeModel), true, true, false, null);
+
+            //this allows undo redo to become active again
+            node.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(new DynamoModel.UndoRedoCommand(DynamoModel.UndoRedoCommand.Operation.Redo));
```

Comment:
Can you explain this a bit? We call a command to make undo re-appear?

Reply:
Yes, it seems like the undoRedo recorder isn't refreshing with just updating the "UndoRedoLock", so I am looking at how to do that. So far it _seems_ a recordable command does it. I can try to see how else to raise the property changed or something like that.